crates/sui-core/src/execution_driver.rs

```diff
@@ -66,7 +66,7 @@ pub async fn execution_process(
             return;
         };
 
-        let epoch_store = authority.epoch_store();
+        let epoch_store = authority.load_epoch_store_one_call_per_task();
```

Comment:
I can't figure out if this is correct or not. We seem to load the epoch store every time we get a new cert. this seems contrary to the new name of this function.

Ideally execution_driver would be a per-epoch task. Its ok if that is too much work for this PR.

Reply:
Yeah added a TODO in this PR